src/octopeer-github/content/ContentController.ts

```diff
@@ -105,7 +105,10 @@ class ContentController {
         this.unhookFromDOM();
         this.hookSemanticToDOM(database);
         this.hookTrackersToDOM(database);
-        this.hookMutationObserverToDOM();
+
+        if (Options.get("dataHTML")) {
```

Comment:
Never mind, I just noticed your changes to the get function. But you can consider to cherry pick my commit on backend changes, so that you can use `Options.get(Options.SOMETHING)`.